draft-ietf-quic-multipath.md

```diff
@@ -160,18 +160,22 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT",
 {{!RFC2119}} {{!RFC8174}} when, and only when, they appear in all
 capitals, as shown here.
 
-We assume that the reader is familiar with the terminology used in
-{{QUIC-TRANSPORT}}. When this document uses the term "path", it refers
+This document uses the terminology defined in {{QUIC-TRANSPORT}}.
+When this document uses the term "path", it refers
 to the notion of "network path" used in {{QUIC-TRANSPORT}}.
 
-# Transport Handshake and Cryptographic Packet Protection
+Further, this document uses the variable-length integer encoding from {{QUIC-TRANSPORT}}.
+And packet diagrams in this document use the format defined in
+{{Section 1.3 of QUIC-TRANSPORT}} to illustrate the order and size of fields.
+
```

Comment:
I was always warned against starting a sentence with And or But... How about:

The packet diagrams in this document uses the conventions defined  in {{Section 1.3 of QUIC-TRANSPORT}},
including the notation `(i)` to denote variable-length integers, encoded as specified in {{Section 16 of QUIC-TRANSPORT}}.